mappings/net/minecraft/server/network/ServerLoginNetworkHandler.mapping

```diff
@@ -27,6 +27,7 @@ CLASS net/minecraft/class_3248 net/minecraft/server/network/ServerLoginNetworkHa
 	FIELD field_14165 serverId Ljava/lang/String;
 	FIELD field_14166 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_14167 nonce [B
+	FIELD field_29779 SLOW_LOGIN_THRESHOLD I
```

Comment:
```suggestion
	FIELD field_29779 TIMEOUT_TICKS I
```

`SLOW_LOGIN_THRESHOLD` is unclear what will happen if a value exceeds this field imo.